src/app/js/public/facet/sagas.js

```diff
@@ -28,9 +30,18 @@ export function* handleLoadFacetValuesRequest({ payload: { name } }) {
     return yield put(loadFacetValuesSuccess({ name, values }));
 }
 
+export function* clearFacetSaga({ payload: { action } }) {
+    if (action === 'POP') {
```

Comment:
please use a constant instead of a string here

Reply:
No constant exist for this value (it come from react-router-redux)